redis/client.py

```diff
@@ -1822,6 +1822,21 @@ def keys(self, pattern='*'):
         "Returns a list of keys matching ``pattern``"
         return self.execute_command('KEYS', pattern)
 
+    def lmove(self, first_list, second_list, src="LEFT", dest="RIGHT"):
+        """
+        Atomically returns and removes the first/last element of a list,
+        pushing it as the first/last element on the destination list.
+        Returns the element being popped and pushed.
+        """
+        params = [first_list, second_list, src, dest]
+        return self.execute_command("LMOVE", *params)
+
+    def blmove(self, first_list, second_list, src="LEFT", dest="RIGHT"):
+        """
+        Blocking version of lmove.
+        """
+        return self.lmove(first_list, second_list, src, dest)
```

Comment:
Why are you calling lmove here instead of executing blmove? Maybe I don't understand something!